components/search/themes/express.css

```diff
@@ -23,6 +23,9 @@
 		--spectrum-search-border-color-focus-hover: var(--spectrum-gray-900);
 		--spectrum-search-border-color-key-focus: var(--spectrum-gray-900);
 
+		--spectrum-search-background-color-disabled: var(--spectrum-disabled-background-color);
+		--spectrum-search-border-color-disabled: var(--spectrum-disabled-border-color);
```

Comment:
Ok, I'm seeing the issue now. If we set this border-color-disabled to be the disabled-border-color, then it fixes the quiet variant, but introduces a new regression for the default disabled state where it now has a visible border against the background:
![image](https://github.com/user-attachments/assets/d7bb724b-ff09-4d52-aaa4-be4307e5fabc)

Not to mention that maybe Chromatic didn't pick this change up, but I think the border color for disabled quiet in S1 might be too light:
![image](https://github.com/user-attachments/assets/70c0b879-63a6-48c3-9d7a-f852c2e0a0b4)

I think what we have to do (at least one option, probably not the only option) is revert this back to what it was before and set an additional quiet selector for S1 and Express.